doc/GETTING_STARTED.md

```diff
@@ -29,87 +25,83 @@ The first step is to install libp2p in your project:
 npm install libp2p
 ```
 
-## Usage
-
-Configuring libp2p should not be a single step, but a continuous task. This includes configuring libp2p to achieve all the project requirements, as well as possible optimizations. Regardless of how you configure libp2p, the top level API will always remain the same.
```

Comment:
I think that we should have at least part of this. Explicitly say that configuring `js-libp2p` should be an iterative process rather than a let's configure everything before running

Reply:
To me this read like, "The libp2p setup is more than a one time thing, you're never done configuring libp2p". It sounds daunting. 

It sounds like you meant something more like:

```
If you're new to libp2p, we recommend configuring your node in stages, as this can make
troubleshooting configuration issues much easier. In this guide, we'll do just that. 
If you're more experienced with libp2p, you may wish to jump to the [Configuration readme](#).
```

Reply:
Yes, I think that is a better way of putting this! Can you change to that in this PR?

Reply:
Added that in.